src/LiveComponent/src/Resources/doc/index.rst

```diff
@@ -1282,6 +1282,116 @@ and ``live_collection_button_delete`` respectively:
+            ],
+        ]
+    ])
+    // ...
+
+If you want more control over how each row is rendered you can override the blocks
+related to the ``LiveCollectionType``. This works the same way as for `the traditional
+collection type`_, but you should use ``live_collection`` and ``live_collection_entry`` as
+prefixes instead.
+
+Let's continue our previous example and customize the rendering of the blog post comments form.
+By default the add comment button is placed after the comments, let's move it before them.
+
+.. code-block:: twig
+
+    {%- block _blog_post_form_comments_widget -%}
+        {%- if button_add is defined and not button_add.rendered -%}
+            {{ form_row(button_add) }}
+        {%- endif -%}
+        {{ block('form_widget') }}
+    {%- endblock -%}
+
+Now add a div around each row:
+
+.. code-block:: twig
+
+    {%- block _blog_post_form_comments_entry_row -%}
+        <div>
+            {{ block('form_row') }}
+            {%- if button_delete is defined and not button_delete.rendered -%}
+                {{ form_row(button_delete) }}
+            {%- endif -%}
+        </div>
+    {%- endblock -%}
+
+.. note::
+
+    The ``LiveCollectionType`` under the hood adds a ``button_add`` and
+    a ``button_delete`` field to the form in a special way. These fields
+    are not added as regular form fields, so they are not part of the form
+    tree, but only the form view. The ``button_add`` is added to the
+    collection view variables and a ``button_delete`` is added to each
+    item view variables.
+
+As an another example, now let's create a general bootstrap 5 theme for the live
+collection type, rendering every item in a table row:
+
+.. code-block:: twig
+
+    {%- block live_collection_widget -%}
+        <table class="table table-borderless form-no-mb">
+            <thead>
+            <tr>
+                {% for child in form|last %}
+                    <td>{{- form_label(child) -}}</td>
+                {% endfor %}
+            </tr>
+            </thead>
+            <tbody>
+                {{ block('form_widget') }}
+            </tbody>
+        </table>
+        {%- if skip_add_button|default(false) is same as false and button_add is defined and not button_add.rendered -%}
+            {{ form_widget(button_add, { label: '+ Add Item', class: 'btn btn-outline-primary' }) }}
+        {%- endif -%}
+    {%- endblock -%}
+
+    {%- block live_collection_entry_row -%}
+        <tr>
+            {% for child in form|filter(child => not child.rendered) %}
+                <td>{{- form_row(child, { label: false }) -}}</td>
+            {% endfor %}
+            <td>
+                {{ form_row(button_delete, { label: 'X', attr: { class: 'btn btn-outline-danger' } }) }}
+            </td>
+        </tr>
+    {%- endblock -%}
+
+If you don't want to customize each row but want to put the add button to a different place,
+later in the template, the ``skip_add_button`` parameter can be used.
```

Comment:
To render the add button later in the template, you can skip rendering it initially with ``skip_add_button``,
then render it manually after: